<?xml version="1.0"?>
<launch>

<node pkg="rosbag" type="record" name="rosbag_raposa" args="record -o /home/siar/siar /tf /odom /odom_ticks /arduimu_v3/imu /cmd_vel 
        /front/depth_registered/camera_info 
        /front/depth_registered/image_raw/compressedDepth
/front/depth_registered/image_raw/compressedDepth/parameter_descriptions
/front/depth_registered/image_raw/compressedDepth/parameter_updates
/front/depth_registered/image_raw/compressed/parameter_updates /front/rgb/camera_info /front/rgb/image_raw/compressed /front/rgb/image_raw/compressed/parameter_descriptions /front/rgb/image_raw/compressed/parameter_updates 
        /back/depth_registered/camera_info  
        /back/depth_registered/image_raw/compressedDepth
/back/depth_registered/image_raw/compressedDepth/parameter_descriptions
/back/depth_registered/image_raw/compressedDepth/parameter_updates /back/rgb/camera_info /back/rgb/image_raw/compressed /back/rgb/image_raw/compressed/parameter_descriptions /back/rgb/image_raw/compressed/parameter_updates
        
        /front_left/depth_registered/image_raw/compressedDepth
        /front_left/depth_registered/camera_info  
/front_left/depth_registered/image_raw/compressedDepth/parameter_descriptions
/front_left/depth_registered/image_raw/compressedDepth/parameter_updates
      /front_left/rgb/camera_info /front_left/rgb/image_raw/compressed 
        /front_left/rgb/image_raw/compressed/parameter_descriptions 
        /front_left/rgb/image_raw/compressed/parameter_updates 
      
      /front_right/depth_registered/image_raw/compressedDepth
        /front_right/depth_registered/camera_info  
/front_right/depth_registered/image_raw/compressedDepth/parameter_descriptions
/front_right/depth_registered/image_raw/compressedDepth/parameter_updates 
      /front_right/rgb/camera_info /front_right/rgb/image_raw/compressed 
        /front_right/rgb/image_raw/compressed/parameter_descriptions 
        /front_right/rgb/image_raw/compressed/parameter_updates 
        
        /back_left/depth_registered/camera_info  
        /back_left/depth_registered/image_raw/compressedDepth
/back_left/depth_registered/image_raw/compressedDepth/parameter_descriptions
/back_left/depth_registered/image_raw/compressedDepth/parameter_updates 
      /back_left/rgb/camera_info /back_left/rgb/image_raw/compressed 
        /back_left/rgb/image_raw/compressed/parameter_descriptions 
        /back_left/rgb/image_raw/compressed/parameter_updates 
      
      /back_right/depth_registered/camera_info  
        /back_right/depth_registered/image_raw/compressedDepth
/back_right/depth_registered/image_raw/compressedDepth/parameter_descriptions
/back_right/depth_registered/image_raw/compressedDepth/parameter_updates 
      /back_right/rgb/camera_info /back_right/rgb/image_raw/compressed 
        /back_right/rgb/image_raw/compressed/parameter_descriptions 
        /back_right/rgb/image_raw/compressed/parameter_updates 
      
      
      /up/depth_registered/camera_info  
        /up/depth_registered/image_raw/compressedDepth
/up/depth_registered/image_raw/compressedDepth/parameter_descriptions
/up/depth_registered/image_raw/compressedDepth/parameter_updates 
        /up/rgb/camera_info /up/rgb/image_raw/compressed 
        /up/rgb/image_raw/compressed/parameter_descriptions 
        /up/rgb/image_raw/compressed/parameter_updates 
      
      
      /best_marker /trajectory_marker /costmap_node/costmap
      
      /thermal_camera/camera_info
      /thermal_camera/image_raw
      /flip_image
      
      
        /rssi_nvip_2400
        /gas_info
        /elec_battery_status
        /motor_battery_status
	/siar_status
	/operation_mode
        " />   
        
</launch>

